airbyte-workers/src/main/java/io/airbyte/workers/process/ProcessBuilderFactory.java

```diff
@@ -30,19 +30,27 @@
 
 public interface ProcessBuilderFactory {
 
-  ProcessBuilder create(String jobId, int attempt, final Path jobPath, final String imageName, final String... args) throws WorkerException;
+  ProcessBuilder create(String jobId, int attempt, final Path jobPath, final String imageName, final String entrypoint, final String... args)
```

Comment:
nit: if you have the time, I think it would be helpful to comment this method/interface. totally understand if you don't, since this was uncommented before your change.